crypto/fipsmodule/rand/rand.c

```diff
@@ -395,6 +363,9 @@ void RAND_bytes_with_additional_data(uint8_t *out, size_t out_len,
     uint8_t seed[CTR_DRBG_ENTROPY_LEN];
     int used_cpu;
     rand_get_seed(state, seed, &used_cpu);
+
+    uint8_t add_data_for_reseed[CTR_DRBG_ENTROPY_LEN];
```

Comment:
I doesn't necessarily have to be `CTR_DRBG_ENTROPY_LEN`. Could be, 32 bytes, say. Dunno if the Jitter module can actually produce that specific amount of bytes or if it always produces, say 64 bytes. If it doesn't have the required permission, `CTR_DRBG_ENTROPY_LEN` is the maximum value.

Reply:
Jitter produces entropy in chunks of 32 bytes, but the additional data is generated with `CRYPTO_sysrand` not Jitter so I'm not sure what you mean here?